### ISSUE TITLE
`make xtest`'s Xcode project fails to build

### ISSUE THREAD

**Issue by jfirebaugh** [mapbox/mapbox-gl-native#735]
I often use the `make xtest` Xcode project for interactive debugging, but it's currently not working. Building the "headless" target in `make xtest`'s Xcode project fails. The root of the failure is the "version" dependency, which generates the following error:

```
/usr/local/Cellar/python/2.7.7_2/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python: can't open file '../scripts/build-version.py': [Errno 2] No such file or directory
```

It looks like this regressed in e11c073e518139ca1729ea1e4a76c215b9abca53.

cc @kkaefer @springmeyer

**Comment by jfirebaugh** [comment]
Ping @kkaefer or @springmeyer could you look at this please?

**Comment by springmeyer** [comment]
@jfirebaugh - replicate and 1717d16eea4b fixes things for me - can you give that a try? I think the issue is that gyp's logic to determine the correct relative paths for actions only gets applied to the command and not the arguments (since its not likely easy to determine which arguments are paths).

**Comment by jfirebaugh** [comment]
@springmeyer Doesn't work for me -- I get a similar error in Xcode as that [on Travis](https://travis-ci.org/mapbox/mapbox-gl-native/builds/49408929):

/Users/john/Library/Developer/Xcode/DerivedData/test-gzomlabvyrukyfapzvcyuosafrey/Build/Intermediates/mapboxgl.build/Debug/version.build/Script-3AB89B10FB9E6C08A72D1FDB.sh: line 2: /Users/john/Development/mapbox-gl-native/../scripts/build-version.py: No such file or directory

**Comment by jfirebaugh** [comment]
Looks like this was fixed by build system changes in #772.
